### PR TITLE
Add app_creation_gas_amount for app creation txs

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add app_creation_gas_amount for app creation txs
- Set missing gas_price for chatbot tool script

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1020

### CODE REVIEW

File: db/functions.js [hunk 1]
```diff
@@ -789,6 +789,10 @@ class Functions {
     if (errorCode) {
       return this.returnFuncResult(context, errorCode);
     }
+    let extraGasAmount = 0;
+    if (isEnabledTimerFlag('add_app_creation_gas_amount', context.blockNumber)) {
```
Comment (liayoo) [inline db/functions.js:799]: Is it intended to be fined for failed txs as well? If not, how about moving this to L823?

File: db/functions.js [hunk 2]
```diff
@@ -775,6 +775,12 @@ class Functions {
     return { sanitizedVal, errorCode: null };
   }
 
+  isNonExistingApp(appName) {
+    const appPath = PathUtil.getAccountPath(appName);
```
Comment (liayoo) [inline db/functions.js:779]: How about using getManageAppConfigPath()?

File: db/functions.js (hunk 1 above)
Comment (platfowner) [reply db/functions.js:799]: As you can see, extra gas amount is added in the success case only (line 831 `this.returnFuncResult(context, FunctionResultCode.SUCCESS, extraGasAmount)`), but the extra gas amount is computed here before core business logic including rule, owner, and config settings.

File: db/functions.js (hunk 2 above)
Comment (platfowner) [reply db/functions.js]: Oops, just fixed. thanks!

**Comment by platfowner** [comment]
Just addressed the comments, PTAL~ @liayoo

File: db/functions.js (hunk 1 above)
Comment (liayoo) [reply db/functions.js:799]: Ah, I see. Thanks for the explanation!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the review. Now merging..

**Pull request merged**